src/methods/cloud-functions.ts

```diff
@@ -20,21 +20,24 @@ export function createPubSubCloudFunctions<Data = unknown, Context = unknown>(
 ): CloudFunctionFun {
   const { parseJson, onError, logger } = options;
   return async (req, res): Promise<void> => {
+    const context: Context = req.body;
```

Comment:
not sure why this pass only the `body` instead of the entire request - but I don't wanna change it now